src/docs/stan-reference/examples.tex

```diff
@@ -8174,7 +8175,7 @@ \subsection{Pathological Solutions}
 algebraic solver will not report these cases, as the algorithm stops once it has found
 an acceptable solution. The initial guess will often determine which solution gets found
 first. The degeneracy may be broken by putting additional constraints on the solution.
-For instance, it might make physical sense for a solution to be positive or negative.
+For instance, it might make ``physical sense" for a solution to be positive or negative.
```

Comment:
This needs to be ` ``physical sense'' ` with two apostrophes trailing to render correctly in LaTeX.